src/WpBookingsFrontUi.php

```diff
@@ -59,7 +66,10 @@ public function setup()
             [
                 'wp_bookings_front_ui' => function () {
                     return $this;
-                }
+                },
+                'bookings_front_ui/options_container' => function () {

```

Comment:
Are you quite sure that this is working? AFAIK, keys with separator in them don't work.

Reply:
Yes, it works. I’ve checked before push :slightly_smiling_face: